src/permissions/index.js

```diff
@@ -45,11 +47,19 @@ const permissions = shield(
     fallbackRule: allow,
 
     // Only allow the external errors when running in the local DEV environment
-    allowExternalErrors: process.env.IS_OFFLINE,
+    // OR when there is a CMR_ERROR a user needs to read
+    allowExternalErrors: allowExternalErrorsBoolean,
 
     // `fallbackError` displays a useful message to the user containing a requestId, rather than the default "Not Authorized!" GraphQL Shield error.
     fallbackError: async (thrownThing, parent, args, context) => {
       const { requestId } = context
+      const { extensions } = thrownThing
+      const { code } = extensions
+
+      // Surfaces cmr errors to mmt
+      if (code === 'CMR_ERROR') {
+        allowExternalErrorsBoolean = false
```

Comment:
You can `return` out of this function before you get to the `throw` below this and it will return the error to the client

```suggestion
        return thrownThing
```